name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages.properties

```diff
@@ -6,7 +6,7 @@ AboutText = Portfolio Performance\n\nVersion: {0} ({1})\nPlatform: {2}, {3}\nJav
 
 AboutTextDeveloped = Developed with contributions by
 
-AboutTextDevelopers = simpsus,jahzoo,gynngr,nistude,mwhesse,derari,necoro,alani1,sebasbaumh,juelicher,wsteitz,beluk,hjbflyer,ZfT2,etzelc,flobjective,akoch,bbrach,tobsen987,gubwe,cmaoling,es86de,DominikHolzapfel,sunnylars,jklw,Ragas13,stephanmunich,fbartnitzek,Galgemaennle,jarischaefer,votacom,Al2Klimov,fuchsst,keszocze,mccare,redevening,tomcastro89,christian-wehrheim,derari,Mauin,masterzuso,rbaeumler,doncristobal,steve84,MarkusTiede,quapla,KopolJunam,verybadsoldier,sennahLessert,tzemp,marco-5,debegr92,toastie89,tenjaa,do3meli,AlphaCentauri1983,andre68723,inv-trad,xemwebe,SoftwareKater,Pnda87,chrisaut,chapph,fvollmer,Mattizin,adrienkohlbecker,benutzername-pp,Verdrahter,SimonScholz,tobias-feldmann,bytefactory73,fabeulous,gregorko,thilo20,mpeterson,jdstamp,Alain-CYS,pascalhein,SerayaEryn,Nirus2000,damarvin,fcaputo,chirlu,fabtagon,MrMarvin,christen90,lahr,NicolaiSchmid,fabian-hahn,huch,pleeplop,roshanshariff,kuehnelth,RomanLangrehr,matthesvoss,D4koon,tweimer,mjtic12,mirkoruether,buletti,kasch-code,alvarobp,flywire,wegwart,rumpeltux,shochmuth,OnkelDok,opt-natter,robertron,metal450,Clemens82,pentamassiv,kingbtcvl,vihtap,0mynona0,tquellenberg,jdugge,auchri,hannnees,digedag,rsnitsch,knorx,eztam-,csprunk,icecoldfire,ah4c,ma4nn,davidm3i,kellerkind87,schurteb,vandercorreiajr,rcjan
+AboutTextDevelopers = simpsus,jahzoo,gynngr,nistude,mwhesse,derari,necoro,alani1,sebasbaumh,juelicher,wsteitz,beluk,hjbflyer,ZfT2,etzelc,flobjective,akoch,bbrach,tobsen987,gubwe,cmaoling,es86de,DominikHolzapfel,sunnylars,jklw,Ragas13,stephanmunich,fbartnitzek,Galgemaennle,jarischaefer,votacom,Al2Klimov,fuchsst,keszocze,mccare,redevening,tomcastro89,christian-wehrheim,derari,Mauin,masterzuso,rbaeumler,doncristobal,steve84,MarkusTiede,quapla,KopolJunam,verybadsoldier,sennahLessert,tzemp,marco-5,debegr92,toastie89,tenjaa,do3meli,AlphaCentauri1983,andre68723,inv-trad,xemwebe,SoftwareKater,Pnda87,chrisaut,chapph,fvollmer,Mattizin,adrienkohlbecker,benutzername-pp,Verdrahter,SimonScholz,tobias-feldmann,bytefactory73,fabeulous,gregorko,thilo20,mpeterson,jdstamp,Alain-CYS,pascalhein,SerayaEryn,Nirus2000,damarvin,fcaputo,chirlu,fabtagon,MrMarvin,christen90,lahr,NicolaiSchmid,fabian-hahn,huch,pleeplop,roshanshariff,kuehnelth,RomanLangrehr,matthesvoss,D4koon,tweimer,mjtic12,mirkoruether,buletti,kasch-code,alvarobp,flywire,wegwart,rumpeltux,shochmuth,OnkelDok,opt-natter,robertron,metal450,Clemens82,pentamassiv,kingbtcvl,vihtap,0mynona0,tquellenberg,jdugge,auchri,hannnees,digedag,rsnitsch,knorx,eztam-,csprunk,icecoldfire,ah4c,ma4nn,davidm3i,kellerkind87,schurteb,vandercorreiajr
```

Comment:
Please revert this change here, don't worry, but this label is copyright by @buchen :wink:

:rofl: _An unwritten rule of the developer_ :rofl:

Reply:
As I said, there are multiple places where recent changes are being undone.